spyder/plugins/updatemanager/workers.py

```diff
@@ -193,25 +305,43 @@ class WorkerUpdate(BaseWorker):
     def __init__(self, stable_only):
         super().__init__()
         self.stable_only = stable_only
-        self.latest_release = None
-        self.update_available = False
+        self.asset_info = None
         self.error = None
         self.checkbox = False
```

Comment:
Keep channel attribute
```suggestion
        self.checkbox = False
        self.channel = None
```